### ISSUE TITLE
Selector //table[id=\"proxylisttable\"] doesn't work

### ISSUE THREAD

**Issue by ststeiger** [zzzprojects/html-agility-pack#314]
I'm fetching `https://free-proxy-list.net/` and want to get the proxys from the list. 

The table element has this signature:
`<table class="table table-striped table-bordered" cellspacing="0" width="100%" id="proxylisttable">`

But selector `"//table[id=\"proxylisttable\"]"` doesn't find anything. 
```
string html = null;

    using (System.Net.WebClient wc = new System.Net.WebClient())
    {
        html = wc.DownloadString("https://free-proxy-list.net/");
    } // End Using wc 



HtmlAgilityPack.HtmlDocument doc = new HtmlAgilityPack.HtmlDocument();
doc.LoadHtml(html);

string selector = "//table[id=\"proxylisttable\"]";
selector = "//table[id='proxylisttable']";
selector = "//table";
HtmlAgilityPack.HtmlNode tableNode = doc.DocumentNode.SelectSingleNode(selector);
System.Console.WriteLine(tableNode);
```

If I put `selector = "//table"; `in there, I get a table node with id="proxylisttable". 

Also as soon as I add HtmlAgilityPack via NuGet, I get the following error (but only on Linux or non-Windows systems respectively): 
https://github.com/dotnet/core/issues/2540
```
/root/.dotnet/sdk/2.2.203/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.targets(5,5): Error MSB4018: The "GenerateDepsFile" task failed unexpectedly.
System.TypeLoadException: Could not load type of field 'Microsoft.NET.Build.Tasks.DependencyContextBuilder:_filteredPackages' (12) due to: Could not resolve type with token 01000027 from typeref (expected class 'NuGet.Packaging.Core.PackageIdentity' in assembly 'NuGet.Packaging, Version=5.0.0.6, Culture=neutral, PublicKeyToken=31bf3856ad364e35') assembly:NuGet.Packaging, Version=5.0.0.6, Culture=neutral, PublicKeyToken=31bf3856ad364e35 type:NuGet.Packaging.Core.PackageIdentity member:(null)
  at Microsoft.NET.Build.Tasks.TaskBase.Execute () [0x00000] in <c5e0e1dbe29a4fae9ad829d4b52ccc8b>:0 
  at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute () [0x00029] in <58f0218f988743a48dd7c84cbe933f4e>:0 
  at Microsoft.Build.BackEnd.TaskBuilder.ExecuteInstantiatedTask (Microsoft.Build.BackEnd.ITaskExecutionHost taskExecutionHost, Microsoft.Build.BackEnd.Logging.TaskLoggingContext taskLoggingContext, Microsoft.Build.BackEnd.TaskHost taskHost, Microsoft.Build.BackEnd.ItemBucket bucket, Microsoft.Build.BackEnd.TaskExecutionMode howToExecuteTask) [0x001f6] in <58f0218f988743a48dd7c84cbe933f4e>:0  (MSB4018) (GeoApis)
```

For reproduction, clone [this](https://github.com/ststeiger/PlotlyTest) repository and try to compile it on Linux with MonoDevelop or JetBrains Rider  (sub-project GeoAPIs).
Sadly it works on Windows.

**Comment by ststeiger** [comment]
Ah, never mind, i missed the @ in the selector. 
Sometimes, you just need to sleep over it to finally discover the full extent of your stupidity ;)

The SDK issue will be fixed with mono 6.0, which isn't yet released as stable. 
https://rider-support.jetbrains.com/hc/en-us/articles/360004180039

So closing this issue.

**Comment by JonathanMagnan** [comment]
Great :) Thank for letting us know.
